app/api/api_v1/endpoints/auth_router.py

```diff
@@ -0,0 +1,47 @@
+from typing import List
+from pydantic import BaseModel
+from fastapi import APIRouter, Request, Depends
+from sqlalchemy.orm import Session
+from app.schemas import RefreshTokenSchema, UserRoleSchema
+from app.services import UserService, JwtService
+from app.core.dependencies import get_db, PermissionDependency, RequireLoginPermission
+
+router = APIRouter()
+
+class LoginBody(BaseModel):
+    onyen: str
+    password: str
+
+class RefreshBody(BaseModel):
+    refresh_token: str
+    
+
+@router.post("/login", response_model=RefreshTokenSchema)
+async def login(
+    *,
+    db: Session = Depends(get_db),
+    login_body: LoginBody
+):
```

Comment:
```suggestion
async def login(*, db: Session = Depends(get_db), login_body: LoginBody):
```
This is only 73 chars long. If it's less than 120 I think we shouldn't wrap, as it starts to actually hurt visibility instead of help.

Reply:
Same for other places below and in other files

Reply:
I disagree specifically for API endpoints. In general, we should be wrapping at 120 characters. However, I think that every endpoint should be wrapped regardless of length. The arguments are very contrived; you can have query params, url path params, JSON body params, form params, dependency injections, request context, etc. all being passed in through the endpoint's arguments, as well as the *args asterisk catch-all, so separating them by line makes it more clear what is actually happening in the endpoint.

Reply:
As long as we do it everywhere I guess that's fine.